### PR TITLE
Portfolio Modals popup overlay

### PR DESCRIPTION
**Pull request by jasonfelice**
### Portfolio Modals Overlay

- Added popup overlay modal to display project details.
- Styled the modal for mobile and desktop versions.
- Added event listeners to the "See Projects" buttons.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -171,6 +174,35 @@ <h1>Profesional Art Printing Data</h1>
...
+                <div class="modal-content">
+                    <img src="images/portfolio/snapshoot-portfolio.png" alt="Portfolio snapshot image">
+                    <div class="modal-lower-half">
+                        <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has
+                            been the industry's standard dummy text ever since the 1500s, when an unknown printer took a
+                            galley of type and scrambled it to make a type specimen book. It has survived not only five
+                            centuries, but also the leap into electronic typesetting, remaining essent</p>
+                        <div class="modal-buttons">
+                            <div class="btns">See Live</div>
+                            <div class="btns">See Source</div>
```
Comment (ShadyShawkat) [inline index.html]: - [ ] Kindly make sure to add the icons on those button elements to match the Figma design.

| Your design | Figma design |
| --- | --- |
| image | image |

File: index.html [hunk 2]
```diff
@@ -171,6 +174,35 @@ <h1>Profesional Art Printing Data</h1>
...
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
```
Comment (ShadyShawkat) [inline index.html]: - [x] Kindly make sure to display the modal properly on mobile view. Your modal is too long and therefore overflows the device's height.

| Your design | Figma design |
| --- | --- |
| image | image |

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @jasonfelice, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js [hunk 3]
```diff
@@ -1,12 +1,35 @@
...
-for (let i = 0; i < document.querySelectorAll('.mobile-menu li a').length; i += 1) {
-  const selectedElement = document.querySelectorAll('.mobile-menu li a')[i];
-  selectedElement.addEventListener('click', () => {
+// for (let i = 0; i < document.querySelectorAll('.mobile-menu li a').length; i += 1) {
+//   const selectedElement = document.querySelectorAll('.mobile-menu li a')[i];
+//   selectedElement.addEventListener('click', () => {
+//     document.querySelector('.mobile-menu').classList.toggle('appear');
+//     document.querySelector('.menu-button').classList.toggle('cross-button');
+//   });
+// }
```
Comment (Whoistolu) [inline script.js:15]: - [ ] [OPTIONAL] Let's remove the commented line of code in order to have a clean ode base.

**Review by Whoistolu — APPROVED** [review]
### Hi @jasonfelice 

Great job implementing the recommended changes 👍🏽 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Whoistolu) [reply index.html]: All good now 👍🏽

File: index.html (hunk 2 above)
Comment (Whoistolu) [reply index.html]: All good now 👍🏽

**Pull request merged**